### DISCUSSION TITLE
Python on Mac Sonoma 14.6.1 trouble with Notify

### DISCUSSION THREAD

**Discussion by julio-liriano** [simpleble/simpleble#335] [Q&A]
Hi! 
I am having some trouble getting "peripheral.notify" working for my setup. I have a "notify" characteristic running on an ESP32C3 board. When I use an nRF dongle, through nRF Connect, I am able to subscribe to it and get periodic updates from that characteristic. 

When I try to do the same in Python on my Mac using simplepyble, I get the following error:
Operation Failed: Characteristic C2022573-7B02-47F4-B9DA-4F4129509FBD Notify/Indicate failed: The attribute could not be found.

Here is the snippet: 
        print("Connecting to device...")
        foundDev.connect()
        time.sleep(1.0)

        for service in foundDev.services():
            print(f"    Service UUID: {service.uuid()}")
            print(f"    Service data: {service.data()}")

            for c in service.characteristics():
                print(f"        Characteristic UUID: {c.uuid()}")
                print(f"        Characteristic Capabilities: {c.capabilities()}")
        
                # enable notify
                print(f"Start Notify on service: {service.uuid()} characteristic: {c.uuid()}")
                try:
                    contents = foundDev.notify(service.uuid(), c.uuid(), lambda data: print(f"Received data: {data}"))
                    time.sleep(5.0)
                    foundDev.unsubscribe(service.uuid(), c.uuid())
                except Exception as e:
                    print(f"Failed to start notify: {e}")

        foundDev.disconnect()

**Comment by kdewald** [comment]
Hey @julio-liriano, thanks for writing!

Could you please reach out to `contact@simpleble.org` so that look into this issue directly?

Thanks!
